### ISSUE TITLE
Question: Is TaskRunResult auto populated?

### ISSUE THREAD

**Issue by poy** [tektoncd/pipeline#2553]
# Expected Behavior

[TaskRunResult](https://godoc.org/github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1#TaskRunResult) is populated after a TaskRun completes.

# Actual Behavior

TaskRunResult isn't in TaskRun's status

# Steps to Reproduce the Problem

1. Create TaskRun
2. Use kubectl to look at TaskRunStatus
3.

# Additional Info

- Kubernetes version:

  **Output of `kubectl version`:**

  ```
  (paste your output here)
  ```

- Tekton Pipeline version:

  **Output of `tkn version` or `kubectl get pods -n tekton-pipelines -l app=tekton-pipelines-controller -o=jsonpath='{.items[0].metadata.labels.version}'`**

v0.10.2

**Comment by sbwsg** [comment]
I don't think that TaskResults were in a user-facing state when the 0.10 branch was released.  We had implemented some of the internal types and changes to Step entrypoints but I'm reasonably sure we hadn't yet exposed them for use.  It's mentioned as a new feature in our 0.11.0-rc1 release notes here: https://github.com/tektoncd/pipeline/releases/tag/v0.11.0-rc1 whereas in older 0.10 we mention only https://github.com/tektoncd/pipeline/pull/1888 which was an internal-only PR.

In 0.11+ Task Results are written by Steps to files at path `/tekton/results/<name of result>` and then the contents of those files are exposed as the Task Results in the TaskRun's Status.
